src/components/CMessageTable.vue

```diff
@@ -64,6 +64,44 @@
             </td>
           </tr>
         </template>
+        <tr class="border text-xs h-10 text-gray-800" v-if="countMessages > 10">
+          <td colspan="1" class="text-left">
+            <div class="ml-5">{{ firstRegister + 1 }}-{{ lastRegister }}</div>
+            <div>
+              <label>
+                <span class="font-bold mt-2 mr-2 ml-5">Rows per page</span>
+                <select v-model.number="sizePage" class="browser-default">
+                  <option v-for="page in listSizePages" :key="page.text" :value="page.value">
+                    {{ page.text }}
+                  </option>
+                  <option key="All" :value="countMessages">All</option>
+                </select>
+              </label>
+            </div>
+          </td>
+          <td colspan="7"></td>
+          <td colspan="1">
+            <div class="m-auto block" v-if="totalPages > 1">
```

Comment:
![image](https://user-images.githubusercontent.com/6851979/85404878-e105dd80-b55f-11ea-9e61-ca350c2e38cb.png)
Maybe we don't need the page count, something like this is enogth right.